tests/actors/rust/wasi-keyvalue-component/src/lib.rs

```diff
@@ -0,0 +1,119 @@
+wit_bindgen::generate!({
+    world: "test-keyvalue",
+    exports: {
+        "wasmcloud:bus/guest": WasiKeyvalueComponent
+    }
+});
+
+use std::str::FromStr;
+
+use crate::exports::wasmcloud::bus::guest::Guest;
+use crate::wasi::{
+    io::streams::write,
+    keyvalue::{
+        atomic::increment,
+        readwrite::{get, set},
+        types::{
+            incoming_value_consume_sync, new_outgoing_value, open_bucket,
+            outgoing_value_write_body, IncomingValue, OutgoingValue,
+        },
+    },
+};
+
+const DEFAULT_BUCKET: &str = ""; // NOTE: custom buckets are not yet supported by many providers
+const DEFAULT_KEY: &str = "test";
+const DEFAULT_TEXT_VALUE: &str = "hello world";
+const DEFAULT_NUMERIC_VALUE: u32 = 42;
+
+struct WasiKeyvalueComponent;
```

Comment:
```suggestion
struct Actor;
```

Just like all other tests